detailed_docs/API/OnDriverDistraction/OnDriverDistraction_TRS.md

```diff
@@ -171,23 +205,33 @@ NOT transfer OnDriverDistraction(DD=OFF) after updating `lockScreenDismissalEnab
     </param>
   </function>
 ``` 
-3. PolicyTable must support new `lock_screen_dismissal_enabled` parameter set by default as 'true' to module_config section of PT.
-```
+
+### 3. PolicyTable 
+PolicyTable must support new `lock_screen_dismissal_enabled` and `lock_screen_dismissal_warning` parameters  
+defined by the following new policy table fields:
+
+```json
 {
     "policy_table": {
         "module_config": {
-            "preloaded_pt": true,
-            "exchange_after_x_ignition_cycles": 100,
-            "exchange_after_x_kilometers": 1800,
-            "exchange_after_x_days": 30,
-            "timeout_after_x_seconds": 60,
-            "seconds_between_retries": [1,
-            5,
-            25,
-            125,
-            625],
+            ...
             "lock_screen_dismissal_enabled": true
-  ```
+        },
+        ...
+        "consumer_friendly_messages" : {
+            ...
+            "lock_screen_dismissal_warning": 
```

Comment:
@KhrystynaDubovyk Actually name of parameter in `messages` section is `LockScreenDismissalWarning`

Reply:
@dboltovskyi, thank you, updated in https://github.com/smartdevicelink/sdl_requirements/pull/126/commits/3a9a8062bc46324deb81ffc1a61540f1fe998bd5